app/views/groups/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<p style="color: green"><%= notice %></p>
+<br/>
+<%= render @group %>
+
+<div>
+  <%= link_to "Back to groups", groups_path, class:"inner-buttons" %>
+
+  <%= button_to "Destroy this group", @group, method: :delete, class:"new-buttons" %>
+</div>

```

Comment:
- [ ] According to the requirements:

![image](https://github.com/joyapisi/budget-app/assets/77069011/037b05fe-c72d-4f28-8813-9c8d422f3de4)

When I click on a category, I'm unable to see the transactions registered under that category. Please make sure to implement it!

|Template|Your design|
|---|---|
| ![image](https://github.com/joyapisi/budget-app/assets/77069011/b22b9d4a-3f5d-4685-b000-0effbc3715bd) | ![image](https://github.com/joyapisi/budget-app/assets/77069011/d05fc0e8-5210-4595-ac69-b5869dde24d0) |

Also, please make sure to include a "Add transaction" button at the bottom of the screen!

Consider making this changes 🤓 🔧

Reply:
@joyapisi  this is not done yet